openapi/components/schemas/GatewayAccountConfig/ParamountCommerce.yaml

```
@@ -44,6 +44,10 @@ allOf:
               - SPORTS_BETTING
               - DIGITAL_REMITTANCE
               - E_SPORTS
+          useDirectDeposit:
+            type: boolean
+            description: Specifies whether to use 1 click direct deposit.
+            default: false
```

Perhaps better to add example too or no need?

```suggestion
            default: false
            example: false
```